### PR TITLE
motis-proxy: Implement route rate limit

### PR DESCRIPTION
**Pull request by jbruechert**
Routing requests can be pretty heavy, so we should stop people from making too many, whether on purpose or not

### CODE REVIEW

**Comment by vkrause** [comment]
Certainly makes sense to have that infrastructure, where exactly to draw the line we have to see once this is actually used more widely I guess.

File: ansible/roles/motis-proxy/files/Rocket.toml [hunk 1]
```diff
@@ -2,9 +2,11 @@
 #
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
+[release]
+ip_header = "X-Forwarded-For"
```
Comment (derhuerst) [inline ansible/roles/motis-proxy/files/Rocket.toml:6]: Just FYI: RFC 7239 replaced the non-standardised `X-Forwarded-*` headers with the [`X-Forwarded` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded#transitioning_from_x-forwarded-for_to_forwarded).

File: ansible/roles/nginx/templates/transitous.conf.j2 [hunk 2]
```diff
@@ -41,6 +41,8 @@ server {
     # MOTIS Proxy
     location /api/ {
         proxy_pass http://localhost:8000/;
+
+        proxy_set_header X-Forwarded-For $remote_addr;
```
Comment (derhuerst) [inline ansible/roles/nginx/templates/transitous.conf.j2:45]: Just FYI: RFC 7239 replaced the non-standardised `X-Forwarded-*` headers with the [`X-Forwarded` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded#transitioning_from_x-forwarded-for_to_forwarded).

Comment (jbruechert) [reply ansible/roles/nginx/templates/transitous.conf.j2:45]: Cool, I'm open to replacing this. It's just editing the nginx config and the Rocket.toml

**Pull request merged**